semperfiwebdesign/all-in-one-seo-pack
# Fix meta box help icon alignment

Issue #1956

## Proposed changes

Fixes an issue where a help icon in the meta box is pushed back from the right side when the social meta module is enabled.

## Types of changes

What types of changes does your code introduce?

- Bugfix (non-breaking change which fixes an issue)

## Checklist


## Testing instructions

Enable the Social Meta module and make sure that the top help icon (right next to the "Upgrade to Pro" link) is aligned properly.

## Further comments

None.
